app/keepers.go

```diff
@@ -171,6 +213,11 @@ func initWasmKeeper(encodingConfig axelarParams.EncodingConfig, keys map[string]
 		wasmDir = filepath.Join(homePath, dbDir, "wasm")
 	}
 
+	// Migrate wasm dir from old path to new path
+	// TODO: Remove this once nodes have migrated
+	oldWasmDir := filepath.Join(homePath, "wasm")
+	funcs.MustNoErr(migrateWasmDir(oldWasmDir, wasmDir))
+
```

Comment:
this doesn't need to be part of the wasm keeper initialization. If it's done earlier this function wouldn't have to know anything about `homePath`

Reply:
it needed to be after the above `wasmDir` construction, but agreed that it's better to move that outside as well